simple-rest-server/remove.go

```diff
@@ -0,0 +1,37 @@
+package main
+
+import (
+	"fmt"
+	"io/ioutil"
+	"strings"
+)
+
+func remove(deleteID string) error {
+	buf, err := ioutil.ReadFile("fake.csv")
```

Comment:
We won't be accessing the file like this - so we can remove your file accessing.